core-api/src/test/java/com/optimizely/ab/event/internal/EventFactoryTest.java

```diff
@@ -335,6 +335,174 @@ public void createConversionEventIgnoresInvalidAndAcceptsValidAttributes() {
         }
     }
 
+    /**
+     * Verify that passing through an list of invalid value attribute causes that attribute to be ignored, rather than
+     * causing an exception to be thrown and passing only the valid attributes.
+     */
+    @Test
+    public void createConversionEventIgnoresInvalidAcceptValidValOfValidAttr() {
```

Comment:
I think you will have to adjust this test to accommodate changes from: https://github.com/optimizely/java-sdk/pull/250

The expectation is that there will be no decisions and `enrich_decisions` should be set to `true`